Client/Frontend/Onboarding/Models/IntroViewModel.swift

```diff
@@ -32,110 +24,47 @@ struct IntroViewModel: OnboardingViewModelProtocol, FeatureFlaggable {
             case .updateSignSync: return "update.signToSync"
             }
         }
-
-        var position: Int {
-            switch self {
-            case .welcome: return 0
-            case .signSync: return 1
-            case .notification: return 2
-            case .updateWelcome: return 0
-            case .updateSignSync: return 1
-            }
-        }
     }
 
     // FXIOS-6036 - Make this non optional when coordinators are used
     var introScreenManager: IntroScreenManager?
 
-    var isFeatureEnabled: Bool {
-        return featureFlags.isFeatureEnabled(.onboardingFreshInstall, checking: .buildOnly)
-    }
-
-    var enabledCards: [IntroViewModel.InformationCards] {
-        let notificationCardPosition = OnboardingNotificationCardHelper().cardPosition
-
-        switch notificationCardPosition {
-        case .noCard:
-            return [.welcome, .signSync]
-        case .beforeSync:
-            return [.welcome, .notification, .signSync]
-        case .afterSync:
-            return [.welcome, .signSync, .notification]
-        }
+    var availableCards: [OnboardingCardViewController]
+    var isDismissable: Bool
+    var profile: Profile
+    private var cardModels: [OnboardingCardInfoModelProtocol]
+
+    // MARK: - Initializer
+    init(
+        introScreenManager: IntroScreenManager? = nil,
+        profile: Profile
+    ) {
+        self.introScreenManager = introScreenManager
+        self.profile = profile
```

Comment:
Can you remove profile? It's not used anymore here.

Reply:
It is used when presenting the sign in vc, so it must be here. It was moved into the view model because it used to live in the viewController, and it shouldn't be there.